Kernel/FileSystem/Ext2FileSystem.cpp

```diff
@@ -855,6 +855,28 @@ ssize_t Ext2FSInode::write_bytes(off_t offset, ssize_t count, const u8* data, Fi
     return nwritten;
 }
 
+u8 Ext2FSInode::posix_file_type(const ext2_dir_entry_2& entry)
```

Comment:
Anyone have a suggestion for a better name? :sweat_smile:

Reply:
Well, we should probably not call it `posix_file_type` since this is not in POSIX :)
How about `file_type_for_directory_entry`?